test/unit/puppet_class_importer_test.rb

```diff
@@ -55,6 +55,16 @@ def setup
     assert !importer.actual_environments.include?("foreman-testing")
   end
 
+  test "should save parameter with different default_value when importing" do
```

Comment:
Since I create the lk itself, once I send it to `update_classes_in_foreman` it's an update and not a new instance and prefeorms the same code as a new env.
I can add another env but it's not really needed.